WooCommerce/WooCommerceTests/ViewRelated/Products/Variations/ProductVariationsViewModelTests.swift

```diff
@@ -0,0 +1,42 @@
+import XCTest
+@testable import WooCommerce
+
+final class ProductVariationsViewModelTests: XCTestCase {
+    func test_more_button_appears_when_product_is_not_empty_and_addProductVariations_feature_is_enabled() {
+        // Arrange
+        let variations: [Int64] = [101, 102]
+        let product = MockProduct().product(variations: variations)
```

Comment:
nit: I would say that the `MockProduct()` is deprecated.
We should try to use `Product().copy(variations: [variations])` instead! :-)

Reply:
Thanks! I updated the tests to use `Product()` instead in https://github.com/woocommerce/woocommerce-ios/pull/3686/commits/679b6c80be4bda0f620b1cdda9bc3e7d4fe0970c.